lib/liquid/static_registers.rb

```diff
@@ -0,0 +1,36 @@
+module Liquid
+  class StaticRegisters
```

Comment:
We might want to implement `each` and include `Enumerable` to make this more like a hash, although iterating over registers is an anti-pattern so I'm OK not doing it.